pkg/common/namespace.go

```diff
@@ -0,0 +1,25 @@
+package common
+
+import "fmt"
+
+type NamespaceMapping int
```

Comment:
looks like the type you declare in the runtime config is string. might be easier to make this `= string` so you can convert easily

Reply:
The type is string so I could either have a mapping from string to enum in order to keep the us of an enum. I currently do this mapping in `namespace_config_provider.go`

Reply:
sure, sounds good